src/main/flight/interpolated_setpoint.c

```diff
@@ -62,25 +62,23 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
 
         setpointDeltaImpl[axis] = setpointSpeed * pidGetDT();
-        
+
         const float ffBoostFactor = pidGetFfBoostFactor();
         float clip = 1.0f;
         float boostAmount = 0.0f;
-        if (ffBoostFactor != 0.0f) {
+        if (axis != FD_YAW && ffBoostFactor != 0.0f) {
             if (pidGetSpikeLimitInverse()) {
-                clip = 1 / (1 + fabsf(setpointAcceleration * pidGetSpikeLimitInverse()));
+                clip = 1 / (1 + (setpointAcceleration * setpointAcceleration * pidGetSpikeLimitInverse()));
                 clip *= clip;
             }
-
             // prevent kick-back spike at max deflection
-            if (fabsf(rawSetpoint) < 0.95f * ffMaxRate[axis] || fabsf(setpointSpeed) > 3.0f * fabsf(prevSetpointSpeed[axis])) {
```

Comment:
The second term allows boost to be applied near max deflection if the stick is quickly accelerated. The purpose is to allow boost on stick return during full rate moves. Given that boost is applied if the raw setpoint is less than 95% deflected anyway there is no way that this term can change behavior around center.

Please add back in.

Reply:
My apologies.  Thanks for the clarification.  I should not have taken it out.  Returned.